Sources/Extensions/Foundation/SignedIntegerExtensions.swift

```diff
@@ -80,6 +69,21 @@ public extension SignedInteger {
 		return (self * n).abs / gcd(of: n)
 	}
 	
+    /// SwifterSwift: Array of digits of integer value.
+    ///
+    ///		180.digits() -> [1, 8, 0]
+    ///
+    /// - Returns: Digits of integer value.
+    public func digits() -> [Self] {
+        var digits: [Self] = []
+        for char in String(self).characters {
+            if let int = IntMax(String(char)) {
+                digits.append(Self(int))
+            }
+        }
+        return digits
+    }
+
 }
 
```

Comment:
I think it is OK to have this as a variable since we are iterating over the length of the integer not the value of it, @SD10 what do you think?

Reply:
Sure, worst case scenario is 19 iterations